pkg/trace/schema/consensus.go

```diff
@@ -90,16 +89,36 @@ const (
 	// following schema:
 	//
 	//  | time  | height | timestamp |
-	BlockTable = "consensus_block_time"
+	BlockTable = "consensus_block"
 
 	// TimestampFieldKey is the name of the field that stores the time at which
 	// the block is proposed.
 	TimestampFieldKey = "timestamp"
+
+	// TxCountFieldKey is the name of the field that stores the number of
+	// transactions in the block.
+	TxCountFieldKey = "tx_count"
+
+	// SquareSizeFieldKey is the name of the field that stores the square size
```

Comment:
[Optional] The square size is the number of shares, it may be mistakenly confused with byte size. Would be good to clarify it in the description.